payu/manifest.py

```diff
@@ -162,7 +162,7 @@ def add_filepath(self, filepath, fullpath, copy=False):
     def add_fast(self, filepath, hashfn=fast_hashes, force=False):
         """
         Bespoke function to add filepaths but set shortcircuit to True, which
-        means only the first calculatable hash will be stored. In this way only
+        means only the first calculable hash will be stored. In this way only
```

Comment:
(seems both are valid but smaller is better :D )

Reply:
Oh! Today-I-Learned! Thanks @marshallward !